en/components/card.md

```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
 </igx-card-actions>
 ```
 
-#### Summary
-In this article we covered a lot of ground with the card component. First, we created a very simple card with text content only. Then added some images to make the card a bit more appealing. Finally, we used some additional Ignite UI for Angular components inside our card, like avatars, buttons and icons, to enrich the experience and add some functionality. The card component is capable of displaying more different layouts worth exploring in the Card Demo in the beginning of this article.
+### Styling
+To get started with styling the card, we need to import the `index` file, where all the theme functions and component mixins live:
+
+```scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+``` 
+Following the simplest approach, we create a new theme that extends the [`igx-card-theme`]({environment:sassApiUrl}/index.html#function-igx-card-theme) and accepts some parameters that style the card's items:
+
+```scss
+$dark-card: igx-card-theme(
+    $background:#fd6bd93a,
+    $header-text-color:#f2f607,
+    $subtitle-text-color:#fa2509,
+    $content-text-color:#0a41f7
+);
+```
+As seen, the `igx-card-theme` exposes some useful parameters for basic styling of its items. 
+
+The last step is to **include** the component's theme.
+
+```scss
+ @include igx-card($dark-card);
+```
+>[!NOTE]
+>If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`:
+
+```scss
+:host {
+     ::ng-deep {
+        @include igx-card($dark-card);
+    }
+}
+```
+#### Defining a color palette
+
+Instead of hardcoding the color values like we just did, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$yellow-color: #e7eb0b;
+$black-color: #f00b1e;
+
+$dark-palette: igx-palette($primary: $black-color, $secondary: $yellow-color);
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the palette. 
+
+```scss
+$dark-card: igx-card-theme(
+    $palette: $dark-palette,
+    $background:#fd6bd93a,
+    $header-text-color:#f2f607,
+    $subtitle-text-color:#fa2509,
+    $content-text-color:#0a41f7
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
+
+#### Using Schemas
+
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/schemas.md). A **schema** is a recipe of a theme.
```

Comment:
The **schemas** hyperlink does not work.

Reply:
Fixed